frontend/apps/filemanager/filemanagerutil.lua

```diff
@@ -29,12 +29,50 @@ function filemanagerutil.abbreviate(path)
     return path
 end
 
--- Purge doc settings in sidecar directory,
+-- Purge doc settings in sidecar directory
 function filemanagerutil.purgeSettings(file)
     local file_abs_path = util.realpath(file)
     if file_abs_path then
         DocSettings:open(file_abs_path):purge()
     end
 end
 
+-- Purge reflowable doc font and page settings
+function filemanagerutil.purgeViewSettings(file)
+    local view_settings = {
+        "font_face",
+        -- crop tab
+        "copt_h_page_margins",
+        "copt_t_page_margin",
+        "copt_b_page_margin",
+        -- pageview tab
+        "copt_line_spacing",
+        "line_space_percent",
+        -- textsize tab
```

Comment:
What is kept then ?
And why not remove visible_pages, page_overlap_style, toc_ticks_ignored_levels, book style tweak ?
I'm not saying these should be deleted, it just feels a bit arbitrary to remove _your_ choice of settings, or _mine_ - and being not really sure of what got reset.
Before looking at your code, from the screenshots and in spite of the wording, I thought this option would remove everything except bookmarks/highlights (ie. a safer Reset settings :)

Reply:
Have you read the Issue related?
It's a real use case I understand well.
Only settings from the bottom menu are reset.

Reply:
I've just read it, and I understand it.
But _this_ use case is just a use case of changing fonts and wanting to reset some font related settings so the new defaults associated to your new preferred default fonts are used instead.
But Word spacing and page margins are not related to changing fonts - so you might do a bit more than some use case would wish.
Or a bit less than some other use case would wish (ie. I messed with style tweaks, Toc ticks, page overlap).

But may be your choice of settings is the most consistent and the one most wished for, so why not. But it's a bit low at the area of use cases it can solve, as it's limited to a single set of settings.

And it still feels a bit convoluted to have to close the book, open another book, go to History, long-press on that book to `Reset page and font settings`, and re-open it.
Ideally, you would have another menu invoked on the current opened book. a set of "sets" of settings as radio buttons, that you would check for what you want remove: it would close the book, clean up settings, and re-open the book.